Revit_Engine/Query/FilterByType.cs

```diff
@@ -38,6 +38,7 @@ public static partial class Query
         /****              Public methods               ****/
         /***************************************************/
 
+        [Deprecated("3.1", "This is a duplicate of BH.Engine.Base.Query.FilterByType method.", typeof(BH.Engine.Base.Query), "FilterByType")]
```

Comment:
Was anywhere in the Revit TK using this method? Does it need converting over to the Base Engine query? Cause I can only see two files changed here?

Reply:
I made an effort to deprecate instead of deleting because this method has been exposed to the UI (it is Revit_Engine). If not that, I would be more than happy to delete it straight away.

Reply:
Ok cool if nothing is using it code wise then that's fine, that's all I wanted to check :smile: